tbtaf/listener/TBTAFListener.py

```diff
@@ -0,0 +1,92 @@
+import logging
+import shutil
+import os
+
+import configparser
+from git import Repo
+
+from listener.gitpushevents import GitPushEvents
+from orchestrator.TBTAFOrchestrator import TBTAFOrchestrator
+
+
+class Listener(object):
+      
+    def check_push_events(self):
```

Comment:
Suggestion: With the consideration of being able to define multiple listener jobs that listens to multiple repos. I would suggest that you define a method parameter to point to a custom listener.properties files and then, if not provided then yes, we could default to listener/listener.properties